cmd/collaborators/update_test.go

```diff
@@ -39,7 +39,7 @@ func TestUpdateCommand(t *testing.T) {
 				appSelectMock.On("TeamAppSelectPrompt").Return(prompts.SelectedApp{App: types.App{AppID: "A123"}, Auth: types.SlackAuth{}}, nil)
 				// Set experiment flag
 				clientsMock.Config.ExperimentsFlag = append(clientsMock.Config.ExperimentsFlag, "read-only-collaborators")
-				clientsMock.Config.LoadExperiments(context.Background(), clientsMock.IO.PrintDebug)
+				clientsMock.Config.LoadExperiments(ctx, clientsMock.IO.PrintDebug)
```

Comment:
note: Here's another example where we're replacing a `context.Background()` with the new, mocked context `ctx`